src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -438,6 +438,18 @@ interface IOptionSettlementEngine {
     //  Protocol Admin
     //////////////////////////////////////////////////////////////*/
 
+    /**
+     * @notice Check if the protocol fee switch is enabled.
+     * @return Whether or not the protocol fee switch is enabled.
+     */
+    function isFeeSwitchEnabled() external view returns (bool);
+
+    /**
+     * @notice Toggles the protocol fee on / off.
+     * @param enabled Whether or not the protocol fee switch should be enabled.
+     */
+    function toggleFeeSwitch(bool enabled) external;
```

Comment:
nit: naming, prefer `setFeeSwitch`, `toggle` implies that isFeeSwitchEnabled ==> !isFeeSwitchEnabled, whereas what we're doing is setting the boolean to the provided value

Reply:
Makes sense. Updating